samples/cpp/benchmark_app/main.cpp

```diff
@@ -630,22 +651,26 @@ int main(int argc, char* argv[]) {
         next_step();
         // output of the actual settings that the device selected
         for (const auto& device : devices) {
-            auto supported_properties = compiledModel.get_property(ov::supported_properties);
+            auto supported_properties = core.get_property(device, ov::supported_properties);
```

Comment:
you 've changed quering the config from the **compiled_model** (where e,g, the HINTS turned into actual settings like #streams, etc) to quering from the **plugin** (whihc works until _currently_ the bencamrk_app uses the set_property for the plugin). Generally this seems to be an incorrect change

Reply:
the auto-batching is fixed on the recent master, so maybe revert this change ( it is the reason for seeing the Auto-Batching params, not the GPUs' as you was trying to debug in the chat)

Reply:
I've just spotted that this code only queries the params names, not the values (that are still coming from the compiled_model)